using-valtimo/forms/creating-forms-in-valtimo.md

```diff
@@ -22,6 +22,44 @@ property `targetKey` is added, it will store submitted values to the reference p
 If a `sourceKey` is provided, it will store submitted values using the reference provided in the `sourceKey`. Unless
 there is a `targetKey` provided. Then the `targetKey` takes priority.
 
+### ZGW Value resolvers
+
+<sup>Available since 11.2.0</sup>
+
+Valtimo contains additional value resolvers for ZGW services:
+- `zaak` (Zaken API)
+- `zaakstatus` (Zaken API & Catalogi API)
+- `zaakobject` (Zaken API)
+
+With the `zaak` prefix, all data inside
+a [zaak response](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/zaken-api/master/src/openapi.yaml#tag/zaken/operation/zaak_create)
+can be used to prefill the form. Some examples:
+
+| Source key         | Type of data                                            | Example                                                                         |
+|--------------------|:--------------------------------------------------------|---------------------------------------------------------------------------------|
+| zaak:url           | A URL reference to the zaak                             | https://example.com/zaken/api/v1/zaken/f4086828-b0b7-4e6c-a0ac-5ca1e44c5b06     |
```

Comment:
I think these tables should be under references instead

Reply:
Moved